docs/features/kubernetes/authentication.md

```diff
@@ -0,0 +1,26 @@
+---
+id: kubernetes-auth
+title: Kubernetes Authentication
+description: Authentication in kubernetes plugin
+---
+
+The authentication process in kubernetes is basically separate from backstage auth, the
+providers are configured so your kubernetes plugin can locate and access the clusters you
+have access to, the providers currently available are categorized in server side auth and
+client side auth, here's the list:
+
+## Server Side Providers
+
+These providers authenticate your application with the clusters, meaning anyone that is
+logged in into your backstage app will be able to access the same clusters, the providers
```

Comment:
```suggestion
These providers authenticate your _application_ with the cluster, meaning anyone that is
logged in into your backstage app will be granted the same access to Kubernetes objects. The providers
```
The italics might not be totally in-style for these docs, but I find it helps to distinguish "application" in this section from "user" in the next one. Also capitalization and splitting up a run-on sentence.